libmscore/edit.cpp

```diff
@@ -764,35 +764,56 @@ void Score::cmdAddTimeSig(Measure* fm, int staffIdx, TimeSig* ts, bool local)
             return;
             }
 
+      auto getStaffIdxRange = [=](Score* score) -> std::pair<int /*start*/, int /*end*/> {
```

Comment:
It’s better to capture only what is used, instead of capturing everything  
```
auto getStaffIdxRange = [this, local](Score* score) -> std::pair<int /*start*/, int /*end*/> {
```

Reply:
And better `(const Score* score)`

Reply:
Thanks, fixed!